### PR TITLE
For reference ticket #1163

### PR DESCRIPTION
**Pull request by JiangWeian**
Configuration added in default.js of 'qidoURLPrefix' and 'wadoURLPrefix' can control the difference between the two values

### CODE REVIEW

File: platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js [hunk 1]
```diff
@@ -66,12 +66,16 @@ function makeSeriesAsyncLoader(
...
-    });
-
+	
+	
+	var options = {
+		url: server.baseUrl,
+        headers: DICOMWeb.getAuthorizationHeader(server),
+		qidoURLPrefix: server.qidoURLPrefix,
+		wadoURLPrefix: server.wadoURLPrefix,
+	};
```
Comment (dannyrb) [inline platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js]: Is this in line with the options DICOMwebClient expects?

We should probably preserve the old configuration if `qidoURLPrefix`, `wadoURLPrefix`, and `baseUrl` are not specified. If we don't, this becomes a major/breaking change.

Mixed casing is a bit confusing: `qidoURLPrefix`, but `url` in `baseUrl` is not capitalized?

Note: incorrect indentation and some lint rules not applied.

File: platform/viewer/public/config/default.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ window.config = {
         wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
         qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
         wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
+		baseUrl: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE',
+	    qidoURLPrefix: 'rs',
+	    wadoURLPrefix: 'rs',
```
Comment (dannyrb) [inline platform/viewer/public/config/default.js]: These were added, but the `wadoRoot` and `qidoRoot` are still specified?

**Comment by JiangWeian** [comment]
@dannyrb ,OK, I will reconsider your question.

**Comment by dannyrb** [comment]
@JiangWeian, that would do the trick.
That change, and a deprecation warning for the old format, if the new keys aren't specified, when we parse the configuration (`OHIF.log.warn`)

File: platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js [hunk 3]
```diff
@@ -66,12 +66,16 @@ function makeSeriesAsyncLoader(
...
-      headers: DICOMWeb.getAuthorizationHeader(server),
-    });
-
+	
+	
+	var options = {
+		url: server.baseUrl,
+        headers: DICOMWeb.getAuthorizationHeader(server),
+		qidoURLPrefix: server.qidoURLPrefix,
+		wadoURLPrefix: server.wadoURLPrefix,
```
Comment (dannyrb) [inline platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js]: Does it make more sense to not have a baseUrl, and instead specify the `qidoRoot` and `wadoRoot`? We even already have keys for these. What value does this more granular option setting buy us?

**Comment by dannyrb** [comment]
Some additional comments here: https://github.com/OHIF/Viewers/issues/1163

We specify:

    A base URL
    QIDO Prefix URL
    WADO Prefix URL
    QIDO Root (Base + prefix)
    WADO Root (Base + prefix)

Can we safely say the base is always the same? Or should we ditch "prefix" and "base" and simply always use the Root key values? Then add deprecation warnings for the other keys until the next major bump.

**Comment by sedghi** [comment]
base has changed, read more here https://github.com/OHIF/Viewers/issues/3477

**Comment by sedghi** [comment]
I'm closing this as not planned because it's related to v2-legacy. Keeping it open might suggest we support fixing issues in v2-legacy. If this problem persists in v3, please open an issue or, even better, provide a fix for v3.

**Pull request closed without merging**